pkg/server/token/token.go

```diff
@@ -39,6 +39,10 @@ type gimletClaims struct {
 }
 
 func (c gimletClaims) Valid() error {
+	if c.Type != "sess" {
```

Comment:
Use the const instead of the string https://github.com/gimlet-io/gimlet/pull/120/files/3109efaf4829ab29c58ce7edf0853cf32525058d..eec000f64daea3b041b3e43e89ba8168f3e21703#diff-4d8c9ce148d20e0eb009ecba550a97c19efb9109717265367d2de05acfef3f93R29